src/index.js

```diff
@@ -13,18 +13,17 @@ exports = module.exports
 const CHUNK_SIZE = 262144
 
 // Use a layout + chunkers to convert a directory (or file) to the layout format
-exports.import = (options, callback) => {
-  // options.path : what to import
+exports.import = (path, dagService, options, callback) => {
```

Comment:
this doesn't make the 'dagService' non optional. we need to check if it is passed and throw if not the case.